testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
+pkgver=1.14.0
+pkgrel=0
+pkgdesc="Client library written in C for MongoDB"
+url="https://github.com/mongodb/mongo-c-driver"
+arch="all"
+license="Apache-2.0"
+makedepends="openssl-dev cyrus-sasl-dev icu-dev py3-sphinx cmake"
+subpackages="${pkgname}-static ${pkgname}-dev ${pkgname}-doc libbson"
+source=${pkgname}-${pkgver}.tar.gz::https://github.com/mongodb/${pkgname}/archive/${pkgver}.tar.gz
+builddir="${srcdir}/${pkgname}-${pkgver}"
```

Comment:
```suggestion

```

Reply:
I not difference between your suggestion and my string.
Please clarify.

Reply:
The value you are assigning is the default value. The line can be removed.

Reply:
After removing this string build is failed.
Please look current build logs.

Reply:
It did not fail because of the removed string, it failed because you changed the order of the subpackages.

```
>>> mongo-c-driver-static*: Running split function static...
>>> mongo-c-driver-static*: Preparing subpackage mongo-c-driver-static...
/usr/bin/abuild: cd: line 2293: can't cd to /home/buildozer/aports/testing/mongo-c-driver/pkg/mongo-c-driver-static: No such file or directory
>>> ERROR: mongo-c-driver-static*: prepare_package failed
>>> ERROR: mongo-c-driver*: prepare_subpackages failed
>>> ERROR: mongo-c-driver: rootpkg failed
```

Reply:
yes, you are right.
fixed, packaging now correct.
I will mark this conversation as resolved.